package.json

```diff
@@ -30,7 +30,7 @@
   },
   "author": "Matteo Collina <hello@matteocollina.com>",
   "engines": {
-    "node": ">=10"
+    "node": ">=14"
```

Comment:
Since Node 10 and 12 is not included in CI, it's not really possible to guarantee that they are still supported.